### ISSUE TITLE
integrading microsft kinect with Tide

### ISSUE THREAD

**Issue by arnoldas500** [BlueBrain/Deflect#212]
Is there a way to integrate microsoft kinect sensor v2 with tide? In a way like display cluster had it?

**Comment by lj-cug** [comment]
Good idea!  I can use Kinect2 in my OpenGL rendering, following the tool: https://nw.tsuda.ac.jp/lec/kinect2/
